play-app/src/Components/Filter/Filter.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { useProudctFilter } from '../../Context/FilterCont'
+import "./Filter.css"
+
+
+const Filter = () => {
+    const {filterState,filterdispatch} = useProudctFilter();
+  return (
+    <div className='asideContaner'>
+          <ul className="aside-filter pl-m flex flex-row">
+                <h3 className="font-l f-m">Filters</h3>
+                <div className="aside-clear f-m font-l mr-s" onClick={() => filterdispatch({type: "clearAll"})}>Clear</div>
+            </ul>
+            <div className="pl-m pt-m mt-m">
+                <h2 className="font-l flex f-m mb-s">price</h2>
+                <div className="aside-productPrice pt-s flex"><span>रु300</span><span>रु4500</span><span>रु9000</span>
+                </div>
+                <input type="range" min="300" max="9000" onChange={(e) => filterdispatch({type: 'slider',payload : e.target.value})} value={filterState.newPrice} className="aside-slider mt-s" />
+            </div>
+            <div className="mt-m pl-m  " >
+                <h3 className=" pb-m flex f-m font-l">Category</h3>
+                <div className='flex flex-column'>
+                <div className='checkboxes'>
```

Comment:
Suggestion: You can use here Map function to render categories instead of repeating code.

Reply:
ok, wil try